src/content/reference/react/useImperativeHandle.md

```diff
@@ -64,7 +64,7 @@ const MyInput = forwardRef(function MyInput(props, ref) {
 });
 ```
 
-With the code above, [a ref to `MyInput` will receive the `<input>` DOM node.](/reference/react/forwardRef#exposing-a-dom-node-to-the-parent-component) However, you can expose a custom value instead. To customize the exposed handle, call `useImperativeHandle` at the top level of your component:
+Dengan kode di atas, [sebuah *ref* yang diteruskan ke komponen `MyInput` akan menerima *DOM node* dari `<input>`.](/reference/react/forwardRef#exposing-a-dom-node-to-the-parent-component) Namun, Anda dapat mengekspos sebuah nilai kustom. Untuk mengkustom penanganan yang diekspos, panggil fungsi `useImperativeHandle` di tingkat atas komponen Anda:
```

Comment:
```suggestion
Dengan kode di atas, [sebuah *ref* yang diteruskan ke komponen `MyInput` akan menerima simpul DOM node dari `<input>`.](/reference/react/forwardRef#exposing-a-dom-node-to-the-parent-component) Namun, Anda dapat mengekspos sebuah nilai kustom. Untuk mengkustom penanganan yang diekspos, panggil fungsi `useImperativeHandle` di tingkat teratas komponen Anda:
```

1. Translate `DOM node`. [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium#n)
2. Translate `top` to `teratas`, as I proposed at https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5772140.